src/nvim/buffer.c

```diff
@@ -2141,8 +2141,22 @@ void buflist_list(exarg_T *eap)
 
   for (buf = firstbuf; buf != NULL && !got_int; buf = buf->b_next) {
     /* skip unlisted buffers, unless ! was used */
```

Comment:
The comment could be updated. Perhaps something like `skip unspecified buffers`.